srcpkgs/keynav/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'keynav'
+pkgname=keynav
+version=20180821
```

Comment:
use something like `0.0.20180821` to avoid having to revert all version if they ever tag a release.